Documentation-Review-Process.md

```diff
@@ -0,0 +1,164 @@
+* [Default document review process](#default-document-review-process)
+* [Justification for different process](#justification-for-different-process)
+    * [Users should always expect accurate Documentation](#users-should-always-expect-accurate-documentation)
+    * [Some documentation PRs can be fast-tracked](#some-documentation-prs-can-be-fast-tracked)
+    * [Formal doc reviews can be very time-intensive](#formal-doc-reviews-can-be-very-time-intensive)
+* [Decision Authority](#decision-authority)
+* [Determine if a full document review is required](#determine-if-a-full-document-review-is-required)
+    * [Exempt](#exempt)
+    * [Required](#required)
+* [Requesting a formal document review](#requesting-a-formal-document-review)
+* [Lightweight document review process](#lightweight-document-review-process)
+
+## Introduction
+
+The process for reviewing documentation PRs is slightly different from
+[the general process for reviewing code PRs](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#reviewsPR-Review-Guide.md).
+This document explains the differences.
+
+## The Documentation Team
+
+The Documentation Team is responsible for carrying out a full documentation
+review of a PRs containing documentation changes. Such a review includes
+checking the grammar, structure, consistency, and typographical correctness of
+the documentation changes.
+
+The following GitHub team lists the users who are members of the Documentation Team:
+
+- https://github.com/orgs/kata-containers/teams/documentation/members
+
+## Default document review process
+
+[Project maintainers](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#project-maintainers)
+have the power to approve or reject PRs using the pullapprove tool employed by
+all repositories.
+
+If a Documentation Team review is required, the pullapprove configuration will
+apply and the PR will be blocked until the required number of acks are
+obtained for the documentation changes.
+
+The technical writer from the Documentation Team will review the document
+changes and add comments explaining how the wording should be changed. It is
+then up to the PR creator to apply these changes to their original PR,
+rebasing as necessary to ensure a clean `git(1)` history. Once the creator has
+applied the changes they need to re-push their branch and add a comment to the
+PR asking for a re-review. This process continues indefinitely until the the
+Documentation Team is happy with the final result.
+
+## Justification for different process
+
+The documentation review process is different than the code review process for
+the following reasons:
+
+### Users should always expect accurate Documentation
+
+Just like code changes, some documentation changes have to be merged as soon
+as possible to ensure users have the most current and accurate information.
+
+But documentation changes tend to be more urgent - consider a "Priority 1"
```

Comment:
starting a sentence with 'but'? :-) Also, is that s/consider/considered/ ?
I'm actually having trouble parsing this whole paragraph - can you have a re-read and see if it reads OK for me?

Reply:
Reworded for clarity.